documentation/breaking-changes/upcoming-breaking-changes-desktopvirtualization.md

```diff
@@ -0,0 +1,78 @@
+# Upcoming breaking changes in Azure PowerShell
+
+## Az.DesktopVirtualization
+
+The following cmdlets were affected this release:
+- **Get-AzWvdApplication**
+- **Get-AzWvdApplicationGroup**
+- **Get-AzWvdDesktop**
+- **Get-AzWvdHostPool**
+- **Get-AzWvdHostPoolRegistrationInfo**
+- **Get-AzWvdMsixPackage**
+- **Get-AzWvdPrivateEndpointConnection**
+- **Get-AzWvdRegistrationInfo**
+- **Get-AzWvdScalingPlan**
+- **Get-AzWvdScalingPlanPersonalSchedule**
+- **Get-AzWvdScalingPlanPooledSchedule**
+- **Get-AzWvdSessionHost**
+- **Get-AzWvdUserSession**
+- **Get-AzWvdWorkspace**
+- **New-AzWvdApplication**
+- **New-AzWvdApplicationGroup**
+- **New-AzWvdHostPool**
+- **New-AzWvdMsixPackage**
+- **New-AzWvdRegistraionInfo**
+- **New-AzWvdScalingPlan**
+- **New-AzWvdScalingPlanPersonalSchedule**
+- **New-AzWvdScalingPlanPooledSchedule**
+- **New-AzWvdWorkspace**
+- **Register-AzWvdApplicationGroup**
+- **Send-AzWvdUserSessionMessage**
+- **Unregister-AzWvdApplicationGroup**
+- **Update-AzWvdApplication**
+- **Update-AzWvdApplicationGroup**
+- **Update-AzWvdDesktop**
+- **Update-AzWvdHostpool**
+- **Update-AzWvdMsixPackage**
+- **Update-AzWvdScalingPlan**
+- **Update-AzWvdScalingPlanPersonalSchedule**
+- **Update-AzWvdScalingPlanPooledSchedule**
+- **Update-AzWvdSessionHost**
+- **Update-AzWvdWorkspace**
+### The following enums have been converted to strings:
+- CommandLineSetting
+- ApplicationType
+- ApplicationGroupType
+- IdentityType
+- HostPoolType
+- LoadBalancerType
+- PreferredAppGroupType
+- AgentUpdateType
+- IdentityType
+- PersonalDesktopAssignmentType
+- PublicNetworkAccess
+- RegistrationTokenOperation
+- SkuTier
+- SsoSecretType
+- ScalingHostPoolType
+- ResourceIdentityType
+- SessionHandlingOperation
+- SetStartVmOnConnect
+- StartupBehavior
+- SessionHostLoadBalancingAlogrithm
+- StopHostsWhen
+- RemoteApplicationType
+- SessionHostComponentUpdateType
+- LoadBalancerType
+- PersonalDesktopAssignmentType
+- PreferredAppGroupType
+- DaysOfWeek
+### The following properties have changed from struct to string:
```

Comment:
Are "enum to string" and "struct to string" the same category?

Reply:
> Are "enum to string" and "struct to string" the same category?

should be in same category